### PR TITLE
Feat: Single video page

### PR DESCRIPTION
**Pull request by Rohitprasad83**
## Added Single video page
image
image

### CODE REVIEW

File: src/pages/singlevideo/SingleVideo.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                  <i className="fa-regular fa-thumbs-up"></i> Like
+                </span>
+                <span className="pointer">
+                  <i className="fa-regular fa-thumbs-down"></i>Dislike
+                </span>
+                <span className="pointer">
+                  <i className="fa-regular fa-heart"></i>Add to Watch Later
+                </span>
+                <span className="pointer">
+                  <i className="fa-solid fa-layer-group"></i>Add to Playist
```
Comment (vidhi-thakur) [inline src/pages/singlevideo/SingleVideo.jsx:48]: Just a suggestion: Perhaps you can create a list of single video icons and map the array which would prevent the repetition of code.

**Review by vidhi-thakur — APPROVED** [review]
added one minor suggestion, otherwise the code structure looks fine

**Review by SanketDhabarde — APPROVED** [review]
- Ui looks good
- Code looks clean

File: src/pages/singlevideo/SingleVideo.jsx (hunk 1 above)
Comment (Rohitprasad83) [reply src/pages/singlevideo/SingleVideo.jsx:48]: sure will do it

**Pull request merged**